generators/app/templates/resources/lib/service.py

```diff
@@ -5,18 +5,19 @@
 import logging
 import time
 import xbmc
+import xbmcaddon
 
 
 ADDON = xbmcaddon.Addon()
 logger = logging.getLogger(ADDON.getAddonInfo('id'))
-kodiLogging.config()
 
-if __name__ == '__main__':
+
+def run():
     monitor = xbmc.Monitor()
 
     while not monitor.abortRequested():
         # Sleep/wait for abort for 10 seconds
         if monitor.waitForAbort(10):
             # Abort was requested while waiting. We should exit
             break
-        logger.debug("hello addon! %s" % time.time())
\ No newline at end of file
+        logger.debug("hello addon! %s" % time.time())
```

Comment:
%s operator is being deprecated in favor of str.format in python3. Not sure if it is a good idea to promote the usage of: `hello addon {0}".format(time.time())`